src/AnimationRunners.ts

```diff
@@ -247,3 +249,10 @@ export const loop = (loopConfig: LoopProps) => {
     state.position
   ]);
 };
+
+export const useLoop = (loopConfig: LoopProps) => {
```

Comment:
Since all dependencies have disappeared. What is the purpose of having a hook here? Can you elaborate on your use case

Reply:
Wait, I think I made a huge confusion here.
I thought I needed a `useCode` to execute this kind of thing, but in fact I don`t.

I'll remove this piece of code later and just adjust the doc, sorry about it!